### PR TITLE
Add github issue and PR templates

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
Add GitHub pull request and issue template

## QA
- Check the templates is correct

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+## Issue Template
+
+You can use this template to raise two different types of issue. Please choose the appropriate section and delete the other section.
+
+1. Raise a bug where you can report a problem or inconsistency with Vanilla Framework.
+2. Raise a proposal to add/amend a pattern contained within Vanilla Framework
```
Comment (barrymcgee) [inline .github/ISSUE_TEMPLATE.md]: ...inconsistency with Vanilla _Dashboard Theme_.

Comment (barrymcgee) [inline .github/ISSUE_TEMPLATE.md]: ...contained within Vanilla _Dashboard Theme_

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+## Screenshot
+
+[If relevant, include a screenshot]
+
+---------------------------------------------------------------------------------------
+
+# 2. Pattern proposal/amendment
+
+If you would like to add a new pattern or change an existing pattern in Vanilla Framework, please use this template.
```
Comment (barrymcgee) [inline .github/ISSUE_TEMPLATE.md]: ...existing pattern in _Vanilla Dashboard Theme_

**Comment by anthonydillon** [comment]
Cheers @barrymcgee. Fixed and all the other theme branches too.

**Pull request merged**